Styling

spencer opened the pull request:
Big lump of styling things.

spencer committed:
remove theme, update gemfile and config accordingly.

spencer committed:
extract theme to scss

spencer committed:
Add logo (WIP needs PNG fallback code)

spencer committed:
updated html

spencer committed:
add minima-social-icons

spencer committed:
Delete About Page,
Update code theme to zenburn,
add brand logo, colours , fonts and style things.
(removed nav and various extra styles)

paulanthonywilson commented on Gemfile:
```diff
@@ -10,8 +10,8 @@ source "https://rubygems.org"
-# This is the default theme for new Jekyll sites. You may change this to anything you like.
-gem "minima", "~> 2.0"
+
+gem 'jekyll-seo-tag'
```
Oh, `jekyll-seo-tag` seems like a  nice idea

paulanthonywilson commented on _includes/footer.html:
```diff
@@ -1,35 +1,18 @@
-            {%- endif -%}
-        </ul>
-      </div>
+        <div class="footer-col-wrapper">
```
Yeah, better.

paulanthonywilson reviewed (changes requested):
CR: 

There's something awkward about the font. It's very thing, almost like we're running out of toner.

<img width="679" alt="capto_capture 2019-01-04_02-50-41_pm" src="https://user-images.githubusercontent.com/2986/50693899-4692a100-1030-11e9-9593-2b90bc6e1871.png">

Maybe it's just a bit too small? I don't know.

Also the code styling _is_ harder to read.

Comments seem the same as atoms.

<img width="717" alt="capto_capture 2019-01-04_02-53-05_pm" src="https://user-images.githubusercontent.com/2986/50693977-85c0f200-1030-11e9-9133-b607873e6bd1.png">

Code punctuation (commas, colons, etc..) is very faint.

<img width="578" alt="capto_capture 2019-01-04_02-54-53_pm" src="https://user-images.githubusercontent.com/2986/50694042-ba34ae00-1030-11e9-948e-0eee5bc8c0a2.png">

The original style is much easier to read.

<img width="684" alt="capto_capture 2019-01-04_02-56-28_pm" src="https://user-images.githubusercontent.com/2986/50694121-ee0fd380-1030-11e9-8eeb-7040751af079.png">

spencer committed:
Update font-weights and base-font-size

spencer committed:
Revert to previous syntax-highlighting

paulanthonywilson committed:
Original size, but tweak font colour

Copied from main site; makes the body less harsh. That's what I think I meant by looking like it was running out of toner; gave it a not-anti-aliased sort-of look.

Also the author bit is more grey as I think it looks better that way

 ¯\(°_o)/¯☕️

paulanthonywilson reviewed (changes requested):
Much better with the syntax highlighting. Turns out the font size didn't help (though the weight did). I've made a PR to this PR which I think makes it better. https://github.com/CultivateHQ/bytesize-blog/pull/10
